modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItIndexAvailabilityTest.java

```diff
@@ -77,7 +77,7 @@ void indexIsQueriableRightAfterCreationFutureCompletes2() {
                 + String.format("CREATE INDEX %s ON %s (val)", INDEX_NAME, TABLE_NAME)
         );
 
-        assertQuery(format("SELECT * FROM {} WHERE val = 'test'", TABLE_NAME))
```

Comment:
It's not fully related to your patch, but what is the reason for using an already created table? Why don't we use a new one?

Reply:
don`t catch, this test check that index is available immediately after creation, after my changes index is not figured in plan, thus i force it usage...